### DISCUSSION TITLE
sample fractions

### DISCUSSION THREAD

**Discussion by johnpleid** [vdemichev/DiaNN#339] [Q&A]
Hi to everyone
I am very new with DIA NN 
If I have a sample splitted in two fractions (E.G. phospho enriched LC MSMS and unbound fraction LC MSMS) ho can I set the research to merge the identifications of the two LC MSMS runs?
Thnak u
Gio

**Comment by vdemichev** [comment]
Hi Gio,

It's currently not possible to merge these automatically in DIA-NN. But one can alsways do this in R or Python using whatever custom algorithm is best suitable for the given experiment. For this, it's probably better to use non-normalised data.

Best,
Vadim

**Comment by johnpleid** [comment]
thank you

**Comment by st-pf** [comment]
Dear Vadim, 

is this feature currently planned for any upcoming version? 
Would also be super interested in this directly embedded in DIA-NN. 

Best wishes
Stefan

**Reply by vdemichev** [reply]
Hi Stefan,

No, not planned, as the best way different fractions will need to be merged will depend on the experiment design. E.g. is it best to sum intensities in different fractions or take the maximum? At precursor or protein level? What filtering to apply?

Best,
Vadim

**Comment by Harry-young-0** [comment]
Hi Both,

I just wanted to reopen this discussion to pick your brains. I think I have similar data to what you described Gio, I've got some phospho data that has been fractionated as a first and second wash over TiO2 beads. Idea being that lower affinity ph-peptides will be picked up in the second wash to improve coverage. This however means that there is two fractions per sample in the output table. Did you have any thoughts or recommendations on how to merge these?

Ideally this would become quantitative phosphorylation data between samples but I'm unsure if just summing the intensities of the two fractions is a valid approach? 

Would love to hear your thoughts - I know Fragpipe-LFQDIA and Spectronaut seem to condense this down to a single quant column but I'm struggling to find out exactly how!

Cheers,

Harry

**Comment by vdemichev** [comment]
Hi Harry,

> This however means that there is two fractions per sample in the output table. Did you have any thoughts or recommendations on how to merge these?

How quantitative would this really be? Is this fully automated on a robot? One option would be to combine the two fractions and acquire them in a single run.

>  if just summing the intensities of the two fractions is a valid approach?

In DIA-NN in theory you can indeed sum quantities from different fractions. However, with phospho, the question of localisation confidence would also be relevant. Like OK to sum a peptide with 50% confidence and one with 90%? How confident would the result be? 

Best,
Vadim

**Comment by Harry-young-0** [comment]
Hi Vadim, 

Thanks for this. "How quantitative would this really be" is exactly the issue haha. It's not robot based (though it will be) and
I believe the workflow was designed for DDA analysis, assuming that the low abundance peptides of fraction 2 would be lost if combined. Fraction 2 is less abundant across the board and this comes out in intensity and missing plots. But not horrifically:

<img width="425" height="263" alt="image" src="https://github.com/user-attachments/assets/0492b585-651f-489f-abe9-2ef966689349" />
 
<img width="425" height="263" alt="image" src="https://github.com/user-attachments/assets/9e210151-b8e5-4587-8469-40bb9b5c3403" />


Basically, I'm trying to work out if that second fraction is worth putting on the mass spec for the high replicate repeat of this experiment. And if so, then how best to include it in the follow up quantification. It seems fractionation (at least of this style) makes the quant a little more difficult and I'm wondering if there is a good way to approach it or better just steer clear!

Would exclusively using sums from phosphosites_90.tsv for combining quantification at least get around the second point?


Thanks again!

Harry

**Reply by vdemichev** [reply]
> Basically, I'm trying to work out if that second fraction is worth putting on the mass spec for the high replicate repeat of this experiment.

Is there many extra peptide sequences that are being detected, compared to the first one? If not, maybe just not bother with the second and go first-only?

> Would exclusively using sums from phosphosites_90.tsv for combining quantification at least get around the second point?

Yes, probably will be OKish.

**Reply by Harry-young-0** [reply]
Sorry, just wanted to get the data before my reply.

So detecting some unique phosphopeptides (this is all from _90.tsv) : 

<img width="576" height="355" alt="image" src="https://github.com/user-attachments/assets/fb3b4642-8fa9-446c-9dc8-4c2bbd56a807" />

This was with MBR on though so would you expect that that would improve ID's in Fraction 1 because it found them in Fraction 2? Is that how MBR works in DIANN? :)

Thanks again (sorry this has become user-specific troubleshooting rather than general discussion about fraction handling!)

**Reply by vdemichev** [reply]
OK, I would think those 111 sites are not worth the trouble. Especially given that FDR at 90% confidence may mean most of those can be false.

MBR always helps :) 

Best,
Vadim

**Reply by Harry-young-0** [reply]
Awesome, Thanks for this Vadim, good to chat it through! I'll run them together then use only fraction 1 for downstream quant!

All the best,

Harry
